app/code/community/Ebizmarts/MailChimp/Model/Resource/Ecommercesyncdata.php

```diff
@@ -28,8 +28,8 @@ public function markAllAsModified($id,$type)
         $connection =  $this->_getWriteAdapter();
         $connection->update(
             $this->getMainTable(),
-            ['mailchimp_sync_modified'=>1],
-            ['related_id = ?'=> $id, 'type = ?'=>$type]
+            array ('mailchimp_sync_modified'=>1),
+            array ('related_id = ?'=>$id, 'type = ?'=>$type)
```

Comment:
I think these changes are not part of the issue, the issue is "Fixing incorrect way of translating text in CartController" so what a hell do this change here.
Put this change on a separate issue.